src/components/OverlayManager/Cart/ProductList.tsx

```diff
@@ -2,54 +2,93 @@ import * as React from "react";
 import { TaxedMoney } from "@components/containers";
 import { Thumbnail } from "@components/molecules";
 import { LineI } from "../../CartTable/ProductRow";
+import ReactSVG from "react-svg";
+import editIcon from "../../../images/baseline-edit-24px.svg";
+import cancelIcon from "../../../images/cancel-24px.svg";
+
+enum LocalStorageKeys {
+  Cart = "cart",
+}
 
 const ProductList: React.SFC<{
   lines: LineI[];
   remove(variantId: string): void;
   add(variantId: string, quantity: number): void;
   subtract(variantId: string, quantity: number): void;
-}> = ({ lines, remove, add, subtract }) => (
-  <ul className="cart__list">
-    {lines.map(line => {
-      return (
-        <li key={line.id} className="cart__list__item">
-          <Thumbnail source={line.product} />
-          <div className="cart__list__item__details">
-            <p>{line.product.name}</p>
-            <div className="cart__list__item__details__variant">
-              <span>SKU: {`W1230-CYOHH`}</span>
-              <span>Dimension: {`10"w x 12"h x 24"d`}</span>
-            </div>
-            <div className="cart__list__item__details__pricing">
-              <p>
-                <TaxedMoney taxedMoney={line.pricing.price} />
-              </p>
-              <QauntSelect
-                id={line.id}
-                quantity={line.quantity}
-                add={add}
-                subtract={subtract}
+}> = ({ lines, remove, add, subtract }) => {
+  const [stockQuantity, setStockQuantity] = React.useState({
+    id: null,
+    stock: 0,
```

Comment:
The stock comes from CMS backend, I don't see any reason it should be set to `0` or need an `id`. I think you might be over complicating things here. Please ask me if you have any doubts.